pysnmp/proto/secmod/eso/priv/des3.py

```diff
@@ -8,6 +8,7 @@
 from pysnmp.proto.secmod.rfc3414.priv import base
 from pysnmp.proto.secmod.rfc3414.auth import hmacmd5, hmacsha
 from pysnmp.proto.secmod.rfc3414 import localkey
+from pysnmp.proto.secmod.rfc3414.localkey import hashPassphraseMD5,localizeKeyMD5,hashPassphraseSHA,localizeKeySHA
```

Comment:
Why importing these functions given the whole `localkey` module is already imported?